documentation/src/main/asciidoc/topics/proc_configuring_sql_stores_query.adoc

```diff
@@ -0,0 +1,52 @@
+[id='configuring-sql-cache-stores-query_{context}']
+= Configuring SQL query stores
+
+Add SQL query cache stores so {brandname} can preload your caches from database tables.
+
+.Prerequisites
+
+* For remote caches, copy any database drivers to the `server/lib` directory in your {brandname} Server installation.
+
+.Procedure
+
+. Create a SQL cache store configuration in one of the following ways:
++
+* Declaratively, add the `persistence` element or field then add `query-jdbc-store` with the following schema namespace:
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+xmlns="urn:infinispan:config:store:sql:{schemaversion}"
```

Comment:
@wburns Is this namespace declaration necessary? It is for JDBC string-based and I'd assume so for SQL store too but I haven't been able to configure a store yet.